tech.json

```diff
@@ -486,5 +486,19 @@
     "name": "canQuickCrumbleEscape",
     "requires": [],
     "note":"The combination of a crumble quick drop, and landing on a lower surface and jumping back over the crumble block before it re-forms."
+  },
+  {
+    "name": "canZebetitesSkip",
+    "requires": [ 
+      "Ice",
```

Comment:
I would change that to `canUseFrozenEnemies`. However, there is also a spark-based Zebetite skip which depends on the runway in the adjacent room. I don't believe we can put the requirements for that in a tech.

IMO the ice zeb skip and speed zeb skip should be two separate techs, and the requirements for the speed one will have to be put directly on the room. We may want to put the requirements for the ice one in the room as well for consistency...?